### ISSUE TITLE
A question about the metrics in the subtype classification task

### ISSUE THREAD

**Issue by Haojiang-11** [DearCaat/E2E-WSI-ABMILX#3]
Hello, Authors:

Congratulations on your outstanding work in this paper. It is truly impressive.

I have a question regarding your subtype classification task (e.g., on the BRCA and NSCLC datasets). Could you please clarify whether the performance metric reported in your tables is Accuracy (Acc) or AUC?

The reason I ask is that we have been using the methods mentioned in your paper (such as the two-stage approach for TransMIL and ABMIL) with the pre-extracted features downloaded from your README. In our experiments, we found that our resulting Accuracy scores are very close to the values you reported, but our AUC scores are quite different.

We would be very grateful if you could confirm which metric was used.

Thank you for your time and for this excellent contribution to the community.

**Comment by Haojiang-11** [comment]
**Thank you for your prompt reply.** You mentioned that for the subtyping task, you split the data into a training set and a testing set with a 7:3 ratio, without a validation set.

I was wondering, **what is your strategy for saving the best model in this case**? My previous approach was to save the model that achieved the lowest loss on the validation set. Without a validation set, how do you determine which model to save for testing?

**Comment by DearCaat** [comment]
My saving strategy is based on the optimal AUC on the test set. As far as I know, this seems to be a more common setup?

**Comment by Haojiang-11** [comment]
Thank you for taking your valuable time to answer my questions. I look forward to seeing more of your great work in the future!
